### DISCUSSION TITLE
Why are images rendered so differently ?

### DISCUSSION THREAD

**Discussion by znw-test** [pixijs/pixijs#8899] [Q&A]
I have a picture with a lot of noise. I render it in pixijsv6.2.2 and webgl2. But the rendered result is very different from the image I opened directly.

This is the comparison image.
<img width="1107" alt="different" src="https://user-images.githubusercontent.com/74667824/204245467-5acb1784-7a52-486e-87ca-6456b50d6a98.png">

This is my code.
https://codepen.io/znw-test/pen/oNyMvGg

If you have any ideas, please let me know. Thank you! 🙏

**Comment by SuperSodaSea** [comment]
This may be related to `BaseTexture`'s `scaleMode` and `mipmap` options. You can try to change these options and see what happened.
